CentrED/Tools/BaseTool.cs

```diff
@@ -17,6 +17,7 @@ public abstract class BaseTool : Tool
     protected static int _chance = 100;
     protected bool _pressed;
     protected bool _areaMode;
+    protected bool _altMode;
```

Comment:
Let's call it _topTilesMode, default to true and change it to false when button is pressed

Reply:
ops, missed this comment. I called it _topTilesOnly is it fine anyway?